runtime/drivers/druid/druidsqldriver/druid_api_sql_driver.go

```diff
@@ -40,193 +55,285 @@ type sqlConnection struct {
 
 var _ driver.QueryerContext = &sqlConnection{}
 
-func (c *sqlConnection) QueryContext(ctx context.Context, query string, args []driver.NamedValue) (driver.Rows, error) {
-	dr := newDruidRequest(query, args)
+type coordinatorHTTPCheck struct {
+	c *sqlConnection
+}
+
+var _ retrier.AdditionalTest = &coordinatorHTTPCheck{}
```

Comment:
Move to later in the file – style guide says to sort functions in call order with exported functionality first (i.e. `QueryContext` first)

Reply:
It didn't complain when CI tests were run. Shouldn't we skip it - it's looks good only from one side but it contradicts the rule - place dependent parts closer together to avoid scrolling back and forth.

Reply:
It's not a lint rule, but described in the style guide here: https://github.com/uber-go/guide/blob/master/style.md#function-grouping-and-ordering.

I think it's a good rule to sort declarations by call order – it doesn't feel intuitive to look at a Druid driver and the first 100+ lines of code are internal network retrying logic.

Reply:
But otherwise you'll see the struct you don't know and have to scroll.

Reply:
Moreover I believe if the auto-formatter cannot do it then it's a maintenance burden.

Reply:
Changed.